packages/lesswrong/components/ultraFeed/UltraFeedItemFooter.tsx

```diff
@@ -38,11 +38,20 @@ const UltraFeedEventsDefaultFragmentMutation = gql(`
 `);
 
 const styles = defineStyles("UltraFeedItemFooter", (theme: ThemeType) => ({
+  // Hide footnote poppers/tooltips inside the footer – they appear and stay open on mobile, obscuring buttons
+  '@global': {
+    '@media (pointer: coarse)': {
+      '.LWPopper-root, .LWPopper-default, .LWPopper-tooltip': {
```

Comment:
I think this will disable _all_ poppers, including outside the footer? (Eg the user menu, notifications drawer, etc.)